src/confetti.js

```diff
@@ -435,6 +442,8 @@
       var ticks = prop(options, 'ticks', Number);
       var shapes = prop(options, 'shapes');
       var scalar = prop(options, 'scalar');
+      var wobble = prop(options, 'wobble', Number);
+      var wobbleSpeed = prop(options, 'wobbleSpeed');
```

Comment:
New options that are added to the API should have a description in the README so that users know what they are and how to use them.

Reply:
Ive updated the docs and renamed the prop. That way i dont touch the wobble prop and instead expose it as a boolean option which seems more appropriate.